R/zzz.R

```diff
@@ -10,7 +10,7 @@
 #'
 #' @importFrom yaml yaml.load_file
 .onLoad <- function(libname, pkgname) {
-  .i18_config <<- yaml.load_file(system.file("config.yaml",
+  .i18_config <<- yaml::yaml.load_file(system.file("config.yaml",
```

Comment:
if you prefer prefix taml, then remove `@importFrom`

Reply:
You can try remove this line and `devtools:check()`. 
I added this because of this the warning I recieved.